tsconfig.lib.json

```diff
@@ -13,7 +13,10 @@
     "baseUrl": ".",
     "allowJs": true,
     "checkJs": true,
-    "isolatedModules": true
+    "isolatedModules": true,
+    "paths": {
+      "@placemarkio/polyline": ["node_modules/@placemarkio/polyline/dist/index.d.ts"]
```

Comment:
Why do we need to specify it here explicitly? Why only this package and nothing else?

Reply:
TS complaines that it can't resolve package type declarations because of an incorrect package.json inside of the module, specifying path to type declarations manually resolves the issue

<img width="923" alt="image" src="https://github.com/Fsss126/maplibre-gl-directions/assets/32033964/095e49a1-a65c-468e-ba19-78882dbb5f2c">

Reply:
Didn't notice there was implicit any.

Are they not banned, by the way? Implicit `any`s. Why would the lib compile at all with what we've got at the moment?

Resolved

Reply:
Without strict mode TS just bypassed it as this is banned only in strict mode